### DISCUSSION TITLE
Site Not Found(404)

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22765] [Other Feature Feedback, Questions, & Ideas]
<p>I inputted all my HTML files, CSS files and img folder then tried to go to my page but it said, Site Not Found, with the number 404 on top. Here is my repository: <a href="https://github.com/Indiana13Jones/stormpizza.github.io/tree/master" rel="nofollow noopener">https://github.com/Indiana13Jones/stormpizza.github.io/tree/master</a>.</p>

**Comment by thinkverse** [comment]
<p>Hello. 👋</p>
<p>To activate a GitHub page, go to your repository settings, scroll down to <code>GitHub Pages</code>, set your source to the branch you want to use, in this case, it seems to be the <code>master</code> branch with <code>/root</code> folder which is the default.</p>
<p>After that, you should see a green label above the source heading saying something along the lines of  <code>Your site is published at &lt;URL&gt;</code>.</p>
<p>If you want the site to be your GitHub root, rename your repository and change the username to your username, so <code>Indiana13Jones.github.io</code> in this case.</p>
<p>Sometimes you might also need to add <code>/index.html</code> at the end of the endpoint, I’ve noticed myself it’s kind-of random, sometimes you have too, sometimes you don’t.</p>
<p>Hope that helps. :slightly_smiling_face:</p>

**Comment by ghost** [comment]
<p>Once I do that, my images are not loaded in there.</p>

**Comment by thinkverse** [comment]
<p>It looks like the image you’re trying to fetch isn’t in the repository, the code is looking for <code>img/InsertPicture.png</code> but that image isn’t in the <a href="https://github.com/Indiana13Jones/stormpizza.github.io/tree/HEAD/img" rel="nofollow noopener">img folder</a>.</p>
<p>Modify your code to add the correct image path or add the correct image and that should be fixed. :slightly_smiling_face:</p>

**Comment by ghost** [comment]
<p>The weird thing is, on my local version of the website it has a big picture in the background, and I put it in inside the CSS folder. However it isnt showing up. I know how to fix the other pic now, thanks! Just need help witht that one.</p>

**Comment by thinkverse** [comment, marked as answer]
<p>Your CSS is using <code>background-image: url(../img/stormpizza.png);</code> the <code>../</code> means the CSS will traverse up one directory and then look for the <code>img</code> folder, the problem with that is that your CSS is in the root folder when the CSS is traversing up one folder it won’t find the <code>img</code> folder, try removing that part and see if the image loads. :slightly_smiling_face:</p>

**Comment by ghost** [comment]
<p>That worked! Thanks. One more question, so sorry for bothering you. I have different oages like a video page and the home one, how can I make links to them with the buttons at the top that I have? I have the coding there i just need to know like the url and how I get it.</p>

**Comment by ghost** [comment]
<p>Nevermind, just found out how! Thanks!</p>

**Comment by ghost** [comment]
<p>So sorry to bother you yet again. When I go onto my page, <a href="http://stormpizzayoutube.github.io" rel="nofollow noopener">stormpizzayoutube.github.io</a>, some items moved up. It looks normal in my local version, how can I fix that?</p>

**Comment by gulshansavanth** [comment]
<p>This helped. Thank you <a class="mention" href="/thinkverse">@thinkverse</a></p>
